test/common/http/utility_test.cc

```diff
@@ -359,7 +359,18 @@ TEST(HttpUtility, SendLocalReplyDestroyedEarly) {
     is_reset = true;
   }));
   EXPECT_CALL(callbacks, encodeData(_, true)).Times(0);
-  Utility::sendLocalReply(false, callbacks, is_reset, Http::Code::PayloadTooLarge, "large");
+  Utility::sendLocalReply(false, callbacks, is_reset, Http::Code::PayloadTooLarge, "large", false);
+}
+
+TEST(HttpUtility, SendLocalReplayHeaderReply) {
```

Comment:
typo "Replay". I think maybe you mean "SendLocalReplyHeadRequest" ?

Reply:
yes, spelling mistakes